### DISCUSSION TITLE
Can't install on Windows 10 for MySQL Workbench

### DISCUSSION THREAD

**Discussion by MamaKit** [dracula/dracula-theme#768] [Q&A] [answered]
I have followed the directions on https://draculatheme.com/mysql-workbench, but nothing happens in Workbench. Everything is still blindingly white. Is there somewhere else I need to change code, or something in Workbench Preferences?

**Comment by luxonauta** [comment, marked as answer]
Hi 🙋🏻‍♂️ @MamaKit, how annoying that this is happening. 😥

✅ The ideal would be to create an Issue in the [app repository](https://github.com/dracula/mysql-workbench); So the maintainer (@michaelcaxias) of the same can try to help you.

**Comment by michaelcaxias** [comment]
Hi @MamaKit!

Currently this theme is only able to change the code section, the rest is changed according to your system. If you are using white theme in Ubuntu for example, MySQL will also be white and the same goes for dark theme.

If the code section is not dracula-themed, you can respond to the issue with your Workbench version and I'll check if the theme options have changed.

Thanks!
